src/main/java/core/basesyntax/service/impl/FruitsConsumerCsv.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.FruitsConsumer;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitsConsumerCsv implements FruitsConsumer {
+    private static final String CSV_HEADER = "fruit,quantity";
+    private final String filePath;
+
+    public FruitsConsumerCsv(String filePath) {
+        this.filePath = filePath;
+    }
+
+    @Override
+    public void accept(List<Fruit> fruits) {
+        FileUtils.writeFile(fruitsToCsvRows(fruits), filePath);
+    }
```

Comment:
You have `FruitsConsumer` class with the `accept()` method, that writes data to file. Let's somehow rename the class and method, because it is not obvious from their names what exactly they do.

Reply:
It's implementation of interface method, this one implementation accept fruits to write to csv, but we can ...use another implementation with different destination (write to DB, send by email etc.). About class name,  - supposed it's fruits consumer which write to CSV.